### PR TITLE
Spara/Multi-language Hackathon

### PR DESCRIPTION
**Pull request by spara**
Lee's summary of the multi-language hackathon

### CODE REVIEW

File: themes/default/content/blog/multi-lang-hackathon/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+We had some great ideas, including but not limited to components for a simplified Google Cloud project setup, an Azure web app service container, and an Amazon SageMaker model with a frontend for serving it.
+
+As part of the hackathon, we received some great feedback from all the participants on improving the multi-language authoring experience, including reducing boilerplate code, ways to improve on the schema authoring experience, and documentation suggestions. We’re keen to continue to refine the experience.
+
+Thank you to all the participants! If you’d like to see some of the components in action, you can [watch our demo session](https://youtu.be/U7-eYSpB4o8). We saw the following component demos:
+
+- Run my d#!n container” which demonstrated a component to run a container on any of the major cloud providers.
```
Comment (pierskarsenbarg) [inline themes/default/content/blog/multi-lang-hackathon/index.md:25]:
```suggestion
- "Run my d#!n container” which demonstrated a component to run a container on any of the major cloud providers.
```

Missing quote at the beginning

File: themes/default/content/blog/multi-lang-hackathon/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+<!--more-->
+
+We had a kickoff meeting on Tuesday morning where Lee Briggs walked participants through the [basic steps for bringing up a multi-language component]("https://www.youtube.com/watch?v=_RXvNS5N8A8). If you’re interested in seeing the kickoff meeting, we’ve recorded it and have made it available for viewing on [Pulumi TV](https://youtu.be/Ogg5cs6vPfc). After the walk-through, teams got down to business!
+
+We had some great ideas, including but not limited to components for a simplified Google Cloud project setup, an Azure web app service container, and an Amazon SageMaker model with a frontend for serving it.
+
+As part of the hackathon, we received some great feedback from all the participants on improving the multi-language authoring experience, including reducing boilerplate code, ways to improve on the schema authoring experience, and documentation suggestions. We’re keen to continue to refine the experience.
+
+Thank you to all the participants! If you’d like to see some of the components in action, you can [watch our demo session](https://youtu.be/U7-eYSpB4o8). We saw the following component demos:
```
Comment (pierskarsenbarg) [inline themes/default/content/blog/multi-lang-hackathon/index.md:23]: Did we get any from the community? If so we should draw attention to those

**Review by pierskarsenbarg — CHANGES_REQUESTED** [review]

File: themes/default/content/blog/multi-lang-hackathon/index.md (hunk 2 above)
Comment (spara) [reply themes/default/content/blog/multi-lang-hackathon/index.md:23]: Not aware of any, any one @leezen?

**Review by pierskarsenbarg — APPROVED** [review]

File: themes/default/content/blog/multi-lang-hackathon/index.md (hunk 2 above)
Comment (leezen) [reply themes/default/content/blog/multi-lang-hackathon/index.md:23]: I believe the demo Paul had of GCP project scaffolding was done in conjunction with Simen.

**Review by leezen — APPROVED** [review]

**Pull request merged**